### PR TITLE
docs: Rename svar

### PR DESCRIPTION
**Pull request by hpohekar**
Rename following abbreviations - 

1. SVARData -> SolutionVariablesData
2. SVARService -> SolutionVariablesService
3. svar -> solution_variables
4. svar_data -> solution_variables_data

### CODE REVIEW

**Comment by mkundu1** [comment]
We need to update the new names in the pyconsole code.

**Comment by seanpearsonuk** [comment]
> Rename following abbreviations -
> 
> 1. SVARData -> SolutionVariablesData
> 2. SVARService -> SolutionVariablesService
> 3. svar -> solution_variables
> 4. svar_data -> solution_variables_data

@hpohekar 
(i) Prefer the singular form Variable~s~ in most cases. See the list below.
(ii) Focus on affecting what is publicly visible. SVAR is adequate for private use. I haven't checked the scope of your changes yet.

1. SVARData -> SolutionVariableData
2. SVARService -> SolutionVariableService
3. svar -> solution_variables
4. svar_data -> solution_variable_data

cc: @mkundu1

**Comment by seanpearsonuk** [comment]
> We need to update the new names in the pyconsole code.

Could this become easier if my other comment is considered?

**Comment by seanpearsonuk** [comment]
> > We need to update the new names in the pyconsole code.
> 
> Could this become easier if my other comment is considered?

OK, I had a look at the code. I don't want to dictate exactly how much gets change although there is a distinction between necessary and "less necessary" there. if we are happy to update the console code, then these changes are fine. We can discuss in the upcoming.

File: src/ansys/fluent/core/session_solver.py [hunk 1]
```diff
@@ -116,9 +121,11 @@ def build_from_fluent_connection(self, fluent_connection):
         self._build_from_fluent_connection(fluent_connection)
 
     @property
-    def svar_data(self) -> SVARData:
-        """Return the SVARData handle."""
-        return service_creator("svar_data").create(self.svar_service, self.svar_info)
+    def solution_variables_data(self) -> SolutionVariablesData:
+        """Return the SolutionVariablesData handle."""
+        return service_creator("solution_variables_data").create(
```
Comment (mkundu1) [inline src/ansys/fluent/core/session_solver.py:126]: These strings `"svar"` and `"svar_data"` passed through `service_creator` are internal and renaming can be avoided (and avoid the corresponding pycosole change).

**Comment by seanpearsonuk** [comment]
@hpohekar as mentioned by @prmukherj  we will keep old, public names as deprecated (with warning).

**Comment by hpohekar** [comment]
https://github.com/ansys/pyfluent/pull/2431

**Pull request closed without merging**